Makefile

```
@@ -47,7 +40,9 @@ help:
 	@echo "  install            make and install"
 	@echo "  install-coverage   make and install with C coverage"
 	@echo "  install-req        install documentation and test dependencies"
-	@echo "  install-venv       install in virtualenv"
+	@echo "  install-venv       (deprecated) install in virtualenv"
+	@echo "  lint	       	    run the lint checks"
+	@echo "  lint-fix 	    run black to (mostly) fix lint issues."
```

```suggestion
	@echo "  lint-fix 	        run black to (mostly) fix lint issues."
```
Match up with the other lines.